flax/experimental/nnx/nnx/graph.py

```diff
@@ -861,6 +872,7 @@ def split(
   def split(
     self, node: A, *filters: filterlib.Filter
   ) -> tuple[GraphDef[A], State, tpe.Unpack[tuple[State, ...]]]:
+    """split"""
```

Comment:
maybe we can add some docstrings from [#3907](https://github.com/google/flax/pull/3907/files)